### PR TITLE
Mock account generation improvements

### PR DESCRIPTION
**Pull request by meriadec**
- prevent account from generate negative balance
- ability to specify number of generated operations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "flow-typed": "flow-typed install -s",
-    "build": "rm -rf lib && babel -sd lib src && flow-copy-source src lib",
+    "build": "rm -rf lib && babel --ignore __tests__ -sd lib src && flow-copy-source -i \"__tests__/**\" src lib",
```
Comment (gre) [inline package.json]: :+1:

File: src/mock/account.js [hunk 2]
```diff
@@ -115,10 +122,16 @@ export function genAccount(id: number | string): Account {
...
+
+  let total = 0;
   account.operations = Array(operationsSize)
     .fill(null)
     .reduce(ops => {
       const op = genOperation(account, ops, currency, rng);
+      if (total + op.amount < 0) {
+        op.amount = -op.amount;
+      }
+      total += op.amount;
```
Comment (gre) [inline src/mock/account.js]: how does it work? genOperation don't go back in time as i thought?

Comment (meriadec) [reply src/mock/account.js]: Everytime before a new op is added to list, we ensure that the total balance will be positive. If not, we instead add the `-op.amount`.

Comment (gre) [reply src/mock/account.js]: but genOperation goes back in time: https://github.com/meriadec/ledger-wallet-common/blob/8d899c190a5ec5149fadb72766c0ab36128dcedf/src/mock/account.js#L41-L51

so i'm not sure this works 🤔 balance needs to be calculated reversely

Comment (gre) [reply src/mock/account.js]: let's discuss more tomorrow, but i'm not sure this works 🤔  maybe it does & i totally missed something the other day :o .

**Pull request merged**